Cargo.toml

```diff
@@ -1,30 +1,35 @@
 [package]
 name = "gateway"
-version = "1.0.3"
+version = "2.0.0"
 authors = ["pguenezan <paul@guenezan.me>"]
 edition = "2021"
 
+[features]
+remove_authorization_header = []
+
 [dependencies]
-hyper = { version = "0.14.16", features = ["full"] }
-tokio = { version = "1.16.1", features = ["full"] }
-http-body = "0.4.4"
-jsonwebtoken = "8.0.1"
-serde = { version = "1.0.136", features = ["derive"] }
-serde_json = "1.0.78"
-serde_yaml = "0.9"
+anyhow = "1.0.53"
 bytes = "1.1.0"
+env_logger = "0.11"
+futures = "0.3.21"
+http-body = "1.0"
+http-body-util = "0.1"
+http-serde = "2.1"
+hyper-tungstenite = "0.15"
+hyper-util = { version = "0.1", features = ["client-legacy", "http1", "server"] }
+hyper = { version = "1.4", features = ["full"] }
+jsonwebtoken = "9.3"
+k8s-openapi = { version = "0.23", default-features = false, features = ["v1_31"] }
```

Comment:
Is `v1_31` fine ? It appears that `v1_21` is no longer supported in recent versions of the library.

Reply:
yeah it is fine :+1: